tests/cypress/integration/newfold-labs/wp-module-onboarding/steps/basic-info.cy.js

```diff
@@ -36,106 +41,118 @@ describe('Basic Info Page', function () {
 
     it('Enter a Title and then Check if it reflects elsewhere', () => {
         const title = 'Hello WordPress';
-        var titleBox = cy.get(':nth-child(1) > label > .nfd-input__field');
-        if(titleBox.should('exist')){
-            titleBox.scrollIntoView();
-            titleBox.clear();
```

Comment:
@officiallygod FYI Cypress tests run async so the `if` check fails because `get` does not return the `titleBox` in time. Modified these for the Description and Social Media inputs as well